app/views/games/show.html.erb

```diff
@@ -8,7 +8,17 @@
 <div class="row">
   <div class="col-lg-3 text-center">
     <h4>
-      <%= @game.get_player_one %>      
+      <%= @game.get_player_one %>
+      <% if !white_queenside_rook.nil? && white_king.can_castle?(white_queenside_rook) && current_user == @game.player_one %>

```

Comment:
It might be less confusing to say `white_queenside_rook.present?` instead of `!white_queenside_rook.nil?`

Reply:
Will change that.